src/plots/plots.js

```diff
@@ -2060,8 +2060,20 @@ plots.doAutoMargin = function(gd) {
             Lib.warn('Too many auto-margin redraws.');
         }
     }
+
+    hideOutOfRangeInsideTickLabels(gd);
 };
 
+function hideOutOfRangeInsideTickLabels(gd) {
+    var axList = axisIDs.list(gd, '', true);
+    for(var i = 0; i < axList.length; i++) {
+        var ax = axList[i];
+
+        var hideFn = ax._hideOutOfRangeInsideTickLabels;
```

Comment:
While dragging to pan or zoom I still see the out-of-range labels, but then on mouse up they disappear. Not a big deal, but it would be a bit nicer to never see them.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5275/commits/47c57738c3a52f0ba4c0790cf7a7032e040d1c62.
Resolving...